src/makeCalculator.js

```diff
@@ -3,8 +3,35 @@
 /**
  * @return {object}
  */
+
+
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+        return a - b;
```

Comment:
[CODE KNOWLEDGE] - The `subtract` method should be updated to take only one argument and subtract it from the `result` property, similar to `subtract(number) { this.result -= number; }`.